### ISSUE TITLE
Facing issues in customizing library

### ISSUE THREAD

**Issue by ZoebHusainSheikh** [thoughtbot/TBAnnotationClustering#7]
I need to display 2/more pins as single pin if they are overlapping/touch each other. 

This library has some hard coded value (magic number) which we are not sure how they are calculated (nothing has been defined in doc as well).

FYI: Our custom annotation size is 48*48. 

Please let us know. We are referring to this method:

```
float TBCellSizeForZoomScale(MKZoomScale zoomScale)
{
    NSInteger zoomLevel = TBZoomScaleToZoomLevel(zoomScale);

    switch (zoomLevel) {
        case 13:
        case 14:
        case 15:
            return 64;
        case 16:
        case 17:
        case 18:
            return 32;
        case 19:
            return 16;

        default:
            return 88;
    }
```

![editedclusterscreenshot](https://cloud.githubusercontent.com/assets/2919027/6077978/63474bce-ae1a-11e4-8654-2ebd4cc532ff.png)

```
}
```
#

**Comment by praveenkumar1599** [comment]
Hi,
I am facing the same issue.
Have you found any fix for the same. If so please help me out

**Comment by ZoebHusainSheikh** [comment]
Nope, I used GoogleMap(iOS) SDK in my last project, they have default handling for clustering, also added zip from previous project..may be I have changed something in those files..Good luck

[TBCoordinateQuadTree.zip](https://github.com/thoughtbot/TBAnnotationClustering/files/1127199/TBCoordinateQuadTree.zip)
